CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Changes to Calva Power Tools
 
 ## [Unreleased]
 
+- Improved README to talk about Clay
+- Fix: #3 Uninitialized namespace issue when using Clay commands
```

Comment:
```suggestion
- Fix: [Uninitialized namespace issue when using Clay commands](https://github.com/BetterThanTomorrow/calva-power-tools/issues/3)
```

This makes the link more visible and will make the changelog text “work” in other contexts.